lib/policies/load-balancing.js

```diff
@@ -820,30 +820,42 @@ class DefaultLoadBalancingPolicy extends LoadBalancingPolicy {
  * Validates and sets the local data center to be used.
  * @param {LoadBalancingPolicy} lbp
  * @param {Client} client
+ * @param {HostMap} hosts
  * @private
  */
-function setLocalDc(lbp, client) {
+function setLocalDc(lbp, client, hosts) {
   if (!(lbp instanceof LoadBalancingPolicy)) {
     throw new errors.DriverInternalError('LoadBalancingPolicy instance was not provided');
   }
 
   if (client && client.options) {
     if (lbp.localDc && !client.options.localDataCenter) {
-      client.log('info', 'Local data center \'' + lbp.localDc + '\' was provided as an argument to' +
-        ' DCAwareRoundRobinPolicy. It is more preferable to specify the local data center using \'localDataCenter\'' +
+      client.log('info', 'Local data center \'' + lbp.localDc + '\' was provided as an argument to the' +
+        ' LoadBalancingPolicy. It is more preferable to specify the local data center using \'localDataCenter\'' +
```

Comment:
Suggestion: "...was provided as an argument to the load-balancing policy. It is preferable to specify..."